### PR TITLE
Hub Hackathon Updates

### PR DESCRIPTION
**Pull request by jherlitz-r7**
hackathon updates for:

tr                          
traceroute
trello
trufflehog
try_bro
tshark
tsv
twilio
twitter
typo_squatter
uniq
unshorten
url_expander
urlscan
viper

### CODE REVIEW

File: tshark/plugin.spec.yaml [hunk 1]
```diff
@@ -3,9 +3,8 @@ extension: plugin
 products: [insightconnect]
 name: tshark
 title: TShark
-description: "TShark is a tool for dumping and analyzing network traffic. With the TShark plugin for
-  Rapid7 InsightConnect, users can open PCAP data for further analysis and intel gathering"
-version: 1.0.2
+description: Open and explore PCAP data for further analysis and intel gathering"
```
Comment (tschmidtke-r7) [inline tshark/plugin.spec.yaml]: Extraneous `"`

File: twilio/help.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Description
 
-[Twilio](https://www.twilio.com/) is a cloud communications platform for building SMS, Voice & Messaging applications
+[Twilio](https://www.twilio.com/) is a cloud communications platform for building SMS and Voice & Messaging applications
```
Comment (tschmidtke-r7) [inline twilio/help.md]: Should this be `building SMS, Voice, and Messaging applications`?

Comment (jherlitz-r7) [reply twilio/help.md]: tbh idk, that sounds good to me though

File: tr/help.md [hunk 3]
```diff
@@ -63,6 +64,7 @@ In case an error is raised, make sure that the given expression can be correctly
 
 # Version History
 
+* 2.0.3 - Help updates
```
Comment (jschipp-r7) [inline tr/help.md]: We haven't released since last hackathon. Remove this line as we can release at 2.0.2.

File: viper/plugin.spec.yaml [hunk 4]
```diff
@@ -3,12 +3,12 @@ extension: plugin
 products: [insightconnect]
 name: viper
 title: "Viper"
-description: "The Viper plugins allows you to use Viper's platform to store, search, and analyze files"
+description: Utilize Viper's platform to store, search, and analyze files
```
Comment (jschipp-r7) [inline viper/plugin.spec.yaml]: Make sure this plugin builds with the apostrophe. I'm inclined to say this is now broken.

File: try_bro/help.md [hunk 5]
```diff
@@ -78,7 +78,6 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
-* 1.0.2 - New spec and help.md format for the Hub
```
Comment (jschipp-r7) [inline try_bro/help.md]: We'll want to keep this line

File: traceroute/help.md [hunk 6]
```diff
@@ -69,15 +69,15 @@ Example output:
...
+_This plugin does not contain any triggers._
 
 ### Custom Output Types
 
 _This plugin does not contain any custom output types._
 
 ## Troubleshooting
 
-This plugin does not contain any troubleshooting information.
+_This plugin does not contain any troubleshooting information._
```
Comment (jschipp-r7) [inline traceroute/help.md]: I saw you manually fixing these so I put a mass fix PR.

https://github.com/rapid7/insightconnect-plugins/pull/215
https://github.com/rapid7/insightconnect-plugins/pull/215

File: urlscan/plugin.spec.yaml [hunk 7]
```diff
@@ -5,8 +5,8 @@ name: urlscan
 title: urlscan.io
 vendor: rapid7
 support: community
-status: []
-description: The URLScan plugin uses URLScan to analyze urls for malicious indicators
+status: [] 
+description: Analyze URLs for malicious indicators using URLScan's website scanner
```
Comment (zyoutz-r7) [inline urlscan/plugin.spec.yaml]:
```suggestion
description: Utilize URLScan website scanner to analyze URLs for malicious indicators
```

File: trufflehog/bin/komand_trufflehog [hunk 8]
```diff
@@ -7,7 +7,7 @@ from komand_trufflehog import connection, actions, triggers
 Name = 'TruffleHog'
 Vendor = 'rapid7'
 Version = '1.1.2'
-Description = 'The TruffleHog plugin searches through git repositories for high entropy strings and secrets digging deep into commit history'
+Description = 'Searche through git repositories for high entropy strings and secrets, digging deep into commit history'
```
Comment (zyoutz-r7) [inline trufflehog/bin/komand_trufflehog]:
```suggestion
Description = 'Search through git repositories for high entropy strings and secrets, digging deep into commit history'
```

File: tr/help.md [hunk 9]
```diff
@@ -4,7 +4,8 @@ The Translate plugin uses the `tr` command in UNIX which is a command line utili
 
 # Key Features
 
-* Use the tr command to replace or delete characters
+* Replace characters
```
Comment (zyoutz-r7) [inline tr/help.md]:
```suggestion
* Replace characters in a string
```

File: tr/help.md [hunk 10]
```diff
@@ -4,7 +4,8 @@ The Translate plugin uses the `tr` command in UNIX which is a command line utili
 
 # Key Features
 
-* Use the tr command to replace or delete characters
+* Replace characters 
+* Delete characters
```
Comment (zyoutz-r7) [inline tr/help.md]:
```suggestion
* Delete characters in a string
```

File: twitter/komand_twitter/actions/block/action.py [hunk 11]
```diff
@@ -30,7 +31,7 @@ def run(self, params={}):
 
         # Return False if no object was returned - we can't be certain the block was successful.
         if not blocked_user:
-            raise Exception("No object returned from Twitter. Uncertain of block success. ")
+            raise PluginException(cause="No object returned from Twitter. Uncertain of block success.")
```
Comment (jschipp-r7) [inline twitter/komand_twitter/actions/block/action.py]: This plugin is using Python 2 SDK. It doesn't support PluginException.

Verify all the plugins pass references with `icon-plugin run -c info`

**Pull request merged**